src/modules/get_selfies_to_review/app/get_selfies_to_review_controller.py

```diff
@@ -0,0 +1,44 @@
+from src.shared.domain.entities.student import Student
+from src.shared.domain.entities.selfie import Selfie
+from src.shared.helpers.errors.domain_errors import EntityError
+from src.shared.helpers.errors.usecase_errors import NoItemsFound
+from src.shared.helpers.errors.controller_errors import MissingParameters
+from src.shared.helpers.http.http_models import OK, BadRequest, HttpRequest, HttpResponse, InternalServerError, NotFound
+from .get_selfies_to_review_usecase import GetSelfiesToReviewUsecase
+from .get_selfies_to_review_viewmodel import GetSelfiesToReviewViewmodel
+
+class GetSelfiesToReviewController:
+    def __init__(self, usecase: GetSelfiesToReviewUsecase):
+        self.getSelfiesToReviewUsecase = usecase
+
+    def __call__(self, request: HttpRequest) -> HttpResponse:
+        try:
+            if request.query_params.get('reviewerRa') is None:
+                raise MissingParameters('reviewerRa')
+
+            if request.query_params.get('nSelfies') != None:
+                if type(request.query_params.get('nSelfies')) != str:
+                    raise EntityError('nSelfies')
+                elif not request.query_params.get('nSelfies').isdecimal():
+                    raise EntityError('nSelfies')
+
+
+            args = (request.query_params.get('reviewerRa'), int(request.query_params.get('nSelfies'))) if request.query_params.get('nSelfies') != None else (request.query_params.get('reviewerRa'),)
+            
+            reviews, reviewer = self.getSelfiesToReviewUsecase(*args)
```

Comment:
porque nao passar direto? Essa forma de fazer ficou meio ruim de ler

Reply:
se for por conta de valor padrao, acho melhor usar outra alternativa. Uma possibilidade é nao ter nSelfies como um argumento da funcao, coloca no __call__ do usecase tipo (ra, **kwargs) ai checa se 'nSelfies' está no kwargs e é diferente de None, se nao colocar 10

Reply:
⬇️